pages/preprints.py

```diff
@@ -55,35 +55,38 @@ class PreprintSubmitPage(BasePreprintPage):
 
-    upload_new_preprint_button = Locator(By.CSS_SELECTOR, '#preprint-form-upload > div > div > div > div > div > div:nth-child(1) > button')
-    upload_from_existing_project_button = Locator(By.CSS_SELECTOR, '#preprint-form-upload > div > div > div > div > div > div:nth-child(2) > button')
-    upload_project_selector = Locator(By.CSS_SELECTOR, '#preprint-form-upload .ember-power-select-placeholder')
-    upload_project_selector_input = Locator(By.CSS_SELECTOR, '#preprint-form-upload .ember-power-select-search-input')
+    upload_from_existing_project_button = Locator(By.XPATH, '//button[text()="Select from an existing OSF project"]')
+    upload_project_selector = Locator(By.CSS_SELECTOR, 'span[class="ember-power-select-placeholder"]')
+    upload_project_selector_input = Locator(By.CSS_SELECTOR, 'input[class="ember-power-select-search-input"]')
     upload_project_help_text = Locator(By.CSS_SELECTOR, '.ember-power-select-option--search-message')
     upload_project_selector_project = Locator(By.CSS_SELECTOR, '.ember-power-select-option')
-    upload_existing_project_new_file_button = Locator(By.CSS_SELECTOR, '#preprint-form-upload .fa-cloud-upload')
-    upload_existing_file_button = Locator(By.CSS_SELECTOR, '#preprint-form-upload .fa-th-list')
     upload_select_file = Locator(By.CSS_SELECTOR, '.file-browser-item > a:nth-child(2)')
-    create_new_component_button = Locator(By.CSS_SELECTOR, '#convertExistingOrCreateComponent .fa-plus-circle')
-    convert_existing_component_button = Locator(By.CSS_SELECTOR, '#convertExistingOrCreateComponent .fa-cube')
-    continue_with_this_project_button = Locator(By.CSS_SELECTOR, '.upload-section-block .btn-success')
-    create_new_component = Locator(By.CSS_SELECTOR, '.upload-section-block .btn-default')
-    upload_save_button = Locator(By.CSS_SELECTOR, '.upload-section-block.ember-view.preprint-form-section.cp-Panel.cp-is-open > div > div > button')
-
-    first_discipline = Locator(By.XPATH, '/html/body/div[4]/div[4]/div/div/div/div[1]/section[3]/div/div/div[2]/div[1]/ul/li[1]')
-    discipline_save_button = Locator(By.CSS_SELECTOR, '#preprint-form-subjects .btn-primary')
+    upload_file_save_continue = Locator(By.CSS_SELECTOR, 'div[class="p-t-xs pull-right"] > button[class="btn btn-primary"]')
 
+    basics_license_dropdown = Locator(By.CSS_SELECTOR, 'select[class="form-control"]', settings.LONG_TIMEOUT)
+    basics_universal_license = Locator(By.CSS_SELECTOR, 'select[class="form-control"] > option:nth-child(3)', settings.QUICK_TIMEOUT)
     basics_tags_section = Locator(By.CSS_SELECTOR, '#preprint-form-basics .tagsinput')
     basics_tags_input = Locator(By.CSS_SELECTOR, '#preprint-form-basics .tagsinput input')
     basics_abstract_input = Locator(By.NAME, 'basicsAbstract')
     basics_save_button = Locator(By.CSS_SELECTOR, '#preprint-form-basics .btn-primary')
 
+    discipline_help_text = Locator(By.CSS_SELECTOR, '#preprint-form-subjects .text-smaller', settings.QUICK_TIMEOUT)
```

Comment:
the following locators aren't being used in the test:
`discipline_help_text`
`authors_search_box`
`supplemental_help_text`